index.php

```diff
@@ -175,6 +177,12 @@ public function __construct($baseDir) {
 		require_once $configFileName;
 		$this->configValues = $CONFIG;
 
+		if (php_sapi_name() !== 'cli' && $this->configValues['upgrade.disable-web'] ?? false) {
```

Comment:
```suggestion
		if (PHP_SAPI !== 'cli' && ($this->configValues['upgrade.disable-web'] ?? false)) {
```

Reply:
not sure... my PHP expertise level is inferior compared to @kesselb and @nickvergessen, so not sure which to pick.
battle of the experts ? :smile:

Reply:
It's your choice ;) I would prefer to wrap the second part because it's easier to understand.

Reply:
ah of course. I meant the sapi thing

Reply:
Some years ago I read an article comparing function call vs reading a constant in PHP. Constant was a bit faster and that's what I do since then: Use the constant if possible. That might be outdated nowadays.

Reply:
https://github.com/nextcloud/server/blob/d4d33e3095fb5d0797ec7c3bb8c93c2d1c905ade/lib/base.php#L563 

*cough*